backend/src/main/java/meetup/connect/event/Event.java

```diff
@@ -18,6 +20,18 @@ public class Event {
   private LocalDateTime dateTo;
   private String address;
   private EventType type;
+
+  @ManyToOne
+  @JoinColumn(name = "owner_id", nullable = false)
+  private User owner;
+
+  @ManyToMany(fetch = FetchType.LAZY)
+  @JoinTable(
+      name = "user_event",
+      joinColumns = @JoinColumn(name = "user_id"),
+      inverseJoinColumns = @JoinColumn(name = "event_id"))
+  private Set<User> attendees;
```

Comment:
nit: I would initialize empty relational mappings, making it null-safe https://stackoverflow.com/a/20716407